CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [X.Y.Z] - 2022-MM-DD
+## [X.Y.Z](https://github.com/[INSERT YOUR 'PROJECT/REPO' HERE]/releases/tag/X.Y.X) - 2022-MM-DD
```

Comment:
Hi @jpl-jengelke - I don't think we should include GitHub repository information in the CHANGELOG. The format suggested is based on https://keepachangelog.com/en/1.0.0/, and you may note in their FAQ they've stated to avoid "...non-portable changelog that can only be displayed to users within the context of GitHub". Suggest we leave the CHANGELOG as is and allow projects to customize as needed.